content/docs/command-reference/status.md

```diff
@@ -156,24 +156,24 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
+bar:
+	changed deps:
+        	modified:           bar
         changed outs:
-                not in cache:      foo
+		not in cache:       foo
 foo.dvc:
         changed outs:
-                deleted:      foo
+                deleted:            foo
         changed checksum
 prepare.dvc:
         changed outs:
-                new:      bar
+                new:                bar
         always changed
 ```
 
-This shows that for stage `bar.dvc`, the dependency `foo` and the
-<abbr>output</abbr> `bar` have changed. Likewise for `foo.dvc`, the dependency
-`foo` has changed, but no output has changed.
+This shows that for stage `bar`, the dependency `bar` and the
+<abbr>output</abbr> `foo` have changed. Likewise for `foo.dvc`, the output `foo`
+has changed, but no dependency has changed.
```

Comment:
@jorgeorpinel didn't make it ready for review as many PRs were open.
Currently I have changed indentation in only 3 files. I have kept a tab (8 spaces) for indentation. If this is correct I'll do it for all docs.
Also this example in `status.md` looked outdated so changed this too.

Reply:
Yes you're already working on 2 PRs so let's focus on those please. Or do you have any extra/idle time to do this too?